app/views/home/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= react_component 'App', {
```

Comment:
Sometimes during copy/paste, the punctuation marks are not properly transcribed. You might have to type it to get the key words to register. Did you have any errors on your views?